test/integration/integration_test.go

```diff
@@ -603,12 +604,12 @@ func TestServerAPI(t *testing.T) {
 	log.Info("Check /backup/delete/{where}/{name}")
 	for i := 1; i <= 5; i++ {
 		out, err = dockerExecOut("clickhouse", "bash", "-c", fmt.Sprintf("curl -sL -XPOST 'http://localhost:7171/backup/delete/local/backup_%d'", i))
-		log.Debugf(out)
+		log.Infof(out)
```

Comment:
oh,
look like you just use Rename in file 
instead of just rename inside only your test

rollback it

Reply:
Done